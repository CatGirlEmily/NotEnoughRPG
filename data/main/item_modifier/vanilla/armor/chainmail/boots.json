[
  {
    "function": "minecraft:set_components",
    "components": {
      "minecraft:custom_data": "{CHAINMAIL_BOOTS:true,rarity:true}"
    },
    "conditions": []
  },
  {
    "function": "minecraft:set_lore",
    "entity": "this",
    "lore": [
      "§7Defense: §a+7",
      " ",
      "§a§lUNCOMMON BOOTS"
    ],
    "mode": "replace_all"
  }
]