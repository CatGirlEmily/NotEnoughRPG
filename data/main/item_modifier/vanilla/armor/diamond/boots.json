[
  {
    "function": "minecraft:set_components",
    "components": {

      "minecraft:custom_data": "{DIAMOND_BOOTS:true,rarity:true}"
    },
    "conditions": []
  },
  {
    "function": "minecraft:set_lore",
    "entity": "this",
    "lore": [
      "§7Defense: §a+15",
      " ",
      "§a§lUNCOMMON BOOTS"
    ],
    "mode": "replace_all"
  }
]