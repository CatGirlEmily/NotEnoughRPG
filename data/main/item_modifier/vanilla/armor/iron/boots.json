[
  {
    "function": "minecraft:set_components",
    "components": {

      "minecraft:custom_data": "{IRON_BOOTS:true,rarity:true}"
    },
    "conditions": []
  },
  {
    "function": "minecraft:set_lore",
    "entity": "this",
    "lore": [
      "§7Defense: §a+10",
      " ",
      "§f§lCOMMON BOOTS"
    ],
    "mode": "replace_all"
  }
]