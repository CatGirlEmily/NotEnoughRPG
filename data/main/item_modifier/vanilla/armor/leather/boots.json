[
  {
    "function": "minecraft:set_components",
    "components": {

      "minecraft:custom_data": "{LEATHER_BOOTS:true,rarity:true}"
    },
    "conditions": []
  },
  {
    "function": "minecraft:set_lore",
    "entity": "this",
    "lore": [
      "§7Defense: §a+5",
      " ",
      "§f§lCOMMON BOOTS"
    ],
    "mode": "replace_all"
  }
]