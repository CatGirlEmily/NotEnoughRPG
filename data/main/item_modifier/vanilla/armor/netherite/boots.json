[
  {
    "function": "minecraft:set_components",
    "components": {

      "minecraft:custom_data": "{NETHERITE_BOOTS:true,rarity:true}"
    },
    "conditions": []
  },
  {
    "function": "minecraft:set_lore",
    "entity": "this",
    "lore": [
      "§7Defense: §a+25",
      " ",
      "§5§lEPIC BOOTS"
    ],
    "mode": "replace_all"
  }
]