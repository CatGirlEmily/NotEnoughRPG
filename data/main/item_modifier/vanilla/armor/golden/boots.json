[
  {
    "function": "minecraft:set_components",
    "components": {

      "minecraft:custom_data": "{GOLDEN_BOOTS:true,rarity:true}"
    },
    "conditions": []
  },
  {
    "function": "minecraft:set_lore",
    "entity": "this",
    "lore": [
      "§7Defense: §a+5",
      " ",
      "§f§lCOMMON BOOTS"
    ],
    "mode": "replace_all"
  }
]